<launch>

  <node name="sem_server_node" pkg="sem_server_ros" type="sem_server.py" output="screen" />

</launch>
